app/views/submission_mailer/_generic_confirmation_email.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="col-md-12">
+  <% person_first_name = @person&.first_name.to_s %>
+
+  <div><b>Hello there, <%= person_first_name %></b>,</div>
+  <br>
+  <div>Thank you for your response to <%= Organization.current_organization %>'s <%= @form_name %>!</div>
+
+  <div>
+    <% if @submission.moderation_requested? || @system_settings.dispatch_moderated? %>
```

Comment:
TODO - still need to build out what it means to request moderation. We've got it hinted at on the form designs, but haven't yet worked through what this looks like and the downstream effects.